### PR TITLE
NXP-28953: add isrecord column in document listing

### PR DESCRIPTION
**Pull request by gcarlin**

### CODE REVIEW

File: i18n/messages-fr.json [hunk 1]
```diff
@@ -1042,4 +1043,4 @@
   "workflowAnalytics.averageTaskDurationPerUser.user": "Utilisateur",
   "workflowAnalytics.averageWorkflowDuration.heading": "Durée moyenne des workflows",
   "workflowAnalytics.workflowInitiators.heading": "Initiateurs de workflows"
-}
+}
```
Comment (RSalem07) [inline i18n/messages-fr.json:1046]: a break line at the end of the file is needed.

File: i18n/messages.json [hunk 2]
```diff
@@ -1162,4 +1163,4 @@
   "workflowAnalytics.averageTaskDurationPerUser.user": "User",
   "workflowAnalytics.averageWorkflowDuration.heading": "Average Workflow Duration",
   "workflowAnalytics.workflowInitiators.heading": "Workflow Initiators"
-}
+}
```
Comment (RSalem07) [inline i18n/messages.json:1166]: a break line at the end of the file is needed.

**Comment by gcarlin** [comment]
It should be good now

File: i18n/messages-es-ES.json [hunk 3]
```diff
@@ -345,6 +345,7 @@
   "documentContentView.datatable.header.author": "Autor",
   "documentContentView.datatable.header.coverage": "Cobertura",
   "documentContentView.datatable.header.created": "Creado",
+  "documentContentView.datatable.header.isRecord": "Es un Record",
```
Comment (andreacornaglia) [inline i18n/messages-es-ES.json:348]: es un registro

Comment (gcarlin) [reply i18n/messages-es-ES.json]: Fixed

Comment (gcarlin) [reply i18n/messages-es-ES.json]: Translation also updated on master

**Pull request closed without merging**